dtfs-central-api/src/helpers/escapeRegExp.ts

```diff
@@ -0,0 +1 @@
+export const escapeRegExp = (string: string) => string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
```

Comment:
Do we want to consider using package `escape-string-regexp`? it is already used by about 4 of our dependencies.

Lodash also has this helper, but `dtfs-central-api` is not loading full lodash at the moment.